exampleCourse/questions/element/hiddenHints/server.py

```diff
@@ -0,0 +1,14 @@
+# Hint is Shown if the user clicks Save & Grade
+
+
+def grade(data):
+    if data["submitted_answers"]["c_2"] == 35:
```

Comment:
The question as written currently has no way for anyone to answer it, so the submitted answers dictionary will never get populated. In other words, you need to add a `pl-integer-input` with this answer name set for this to work. You should also include text description in the question about what submitted answer will cause this hint to display.

Reply:
We tried it before submitting the code and it does work. Please see `question.html:12`:

```
<pl-integer-input answers-name="c_2" label="$1 + 1 = $" correct-answer=2></pl-integer-input>
```

So you would like a comment above "if submitted answer is 35" that says "if submitted answer is 35"?

Reply:
Oh whoops, I was looking at the wrong window 🤦🏽 Though there should be some discussion in the question text about what incorrect answer will cause the hint to appear.